modules/configuration/js/configuration_helper.js

```diff
@@ -71,21 +71,24 @@ $(function () {
         e.preventDefault();
 
         var form = $(this).serialize();
-
-        //validate(form);
+        // Clear previous feedback
+        $('.submit-area > label').remove();
 
         $.ajax({
             type: 'post',
             url: loris.BaseURL + '/configuration/ajax/process.php',
             data: form,
             success: function () {
-                var html = "<label>Submitted</label>";
-                $(html).hide().appendTo('.submit-area').fadeIn(500).delay(1000).fadeOut(500)
+                var html = "<label>Update successful</label>";
+                $(html).hide().appendTo('.submit-area').fadeIn(500).delay(1000)
                 $('input[type="reset"]').attr('disabled','disabled');
             },
             error: function(xhr, desc, err) {
                 console.log(xhr);
```

Comment:
We may not want to leave console.log in the frontend. It's similar to leaving print_r in php. There is console.error as well for errors. @HenriRabalais @zaliqarosli suggestions?

Reply:
i think console.error is fine!

Reply:
(don't like console.log though) i think our linting rules allow error, but not log